### PR TITLE
Update local persistent storage and volume scheduling feature gates move to beta

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

**Comment by msau42** [comment]
Oh ok, i'll add the alpha info back then

**Comment by msau42** [comment]
updated

File: docs/reference/feature-gates.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `false` | Alpha | 1.7 | |
 | `MountContainers` | `false` | Alpha | 1.9 | |
 | `MountPropagation` | `false` | Alpha | 1.8 | |
-| `PersistentLocalVolumes` | `false` | Alpha | 1.7 | |
+| `PersistentLocalVolumes` | `true` | Beta | 1.10 | |
+| `PersistentLocalVolumes` | `false` | Alpha | 1.7 | 1.9 |
```
Comment (tengqm) [inline docs/reference/feature-gates.md]: swap line 59 and 60 please

File: docs/reference/feature-gates.md [hunk 2]
```diff
@@ -67,7 +68,8 @@ different Kubernetes components.
 | `SupportIPVSProxyMode` | `false` | Alpha | 1.8 | |
 | `TaintBasedEvictions` | `false` | Alpha | 1.6 | |
 | `TaintNodesByCondition` | `false` | Alpha | 1.8 | |
-| `VolumeScheduling` | `false` | Alpha | 1.9 | |
+| `VolumeScheduling` | `true` | Beta | 1.10 | |
+| `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
```
Comment (tengqm) [inline docs/reference/feature-gates.md]: swap line 71 and 72 please

**Comment by msau42** [comment]
Swapped

**Pull request merged**